test/AWS.jl

```diff
@@ -0,0 +1,867 @@
+function _now_formatted()
+    return lowercase(Dates.format(now(Dates.UTC), "yyyymmddTHHMMSSsssZ"))
```

Comment:
I'm assuming you want a literal `Z` at the end
```suggestion
    return lowercase(Dates.format(now(Dates.UTC), dateformat"yyyymmdd\THHMMSSsss\Z"))
```